providers/base.ts

```diff
@@ -335,6 +336,20 @@ export abstract class ReleaseLookup<Provider extends MetadataProvider, RawReleas
 		};
 	}
 
+	/**
+	 * Shows a warning if the provider found more than the expected result for a lookup.
+	 * Expects a list of URLs pointing to the extra results from the provider.
+	 */
+	protected warnMultipleResults(urls: string[] | URL[]) {
+		const lines = urls.map((url) => `${url} ([lookup](/release?url=${encodeURIComponent(String(url))}))`);
```

Comment:
I like the idea of having a link to initiate a new lookup, it is something which I had also considered doing. My only concern is that the provider implementation now does assumptions on how it is being used, that is how a host-relative link can initiate a lookup on the website.
That is something which I have tried to avoid so far. But I think we can temporarily add this until the providers have a dedicated way to indicate they have multiple results to select from.

Maybe we could use a document-relative link instead which only makes assumptions about the accepted query parameters?
```suggestion
		const lines = urls.map((url) => `${url} ([lookup](?url=${encodeURIComponent(String(url))}))`);
```